t/router/radixtree-host-uri2.t

```diff
@@ -376,3 +376,39 @@ GET /server_port
 Host: test.com
 --- no_error_log
 [error]
+
+
+
+=== TEST 14: inherit hosts from services
+--- yaml_config eval: $::yaml_config
+--- apisix_yaml
+services:
+  - id: 1
+    hosts:
+      - bar.com
```

Comment:
the service hosts here is `bar.com`, but the host in request is `www.foo.com`,how they match?

Reply:
Yes, we test with unmatched. If the host doesn't take effect, the route will match and the response body is "hello1 world".